form/form_collections.rst

```diff
@@ -11,13 +11,8 @@ that Task, right inside the same form.
 
 .. note::
 
-    In this article, it's loosely assumed that you're using Doctrine as your
-    database store. But if you're not using Doctrine, it's all very similar.
-    There are only a few parts of this tutorial that really care about "persistence".
-
-    If you *are* using Doctrine, you'll need to add the Doctrine metadata,
-    including the ``ManyToMany`` association mapping definition on the Task's
-    ``tags`` property.
+    The example presented here is based on Doctrine. However, if you're not
+    using Doctrine, it's all very similar.
```

Comment:
I would remove the second sentence. We cannot assume that it is very similar when not using Doctrine. In any case, I don't think it brings much value to keep this second sentence.

Reply:
Thank you :-) Done!